File: mRemoteV1/CS/Forms/OptionsPages/StartupExitPage.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;
// End of VB project level imports

using mRemoteNG.My;


namespace mRemoteNG.Forms.OptionsPages
{
	public partial class StartupExitPage
		{
			public StartupExitPage()
			{
				InitializeComponent();
			}
public override string PageName
			{
				get
				{
					return Language.strStartupExit;
				}
				set
				{
				}
			}
			
			public override void ApplyLanguage()
			{
				base.ApplyLanguage();
				
				chkSaveConsOnExit.Text = Language.strSaveConsOnExit;
				chkReconnectOnStart.Text = Language.strReconnectAtStartup;
				chkSingleInstance.Text = Language.strAllowOnlySingleInstance;
				chkProperInstallationOfComponentsAtStartup.Text = Language.strCheckProperInstallationOfComponentsAtStartup;
			}
			
			public override void SaveSettings()
			{
				base.SaveSettings();
				
				Settings.SaveConsOnExit = chkSaveConsOnExit.Checked;
				Settings.OpenConsFromLastSession = chkReconnectOnStart.Checked;
				Settings.SingleInstance = chkSingleInstance.Checked;
				Settings.StartupComponentsCheck = chkProperInstallationOfComponentsAtStartup.Checked;
			}
			
			public void StartupExitPage_Load(System.Object sender, EventArgs e)
			{
				chkSaveConsOnExit.Checked = Settings.SaveConsOnExit;
				chkReconnectOnStart.Checked = Settings.OpenConsFromLastSession;
				chkSingleInstance.Checked = Settings.SingleInstance;
				chkProperInstallationOfComponentsAtStartup.Checked = Settings.StartupComponentsCheck;
			}
		}
}

File: mRemoteV1/Forms/OptionsPages/StartupExitPage.vb



Namespace Forms.OptionsPages
    Public Class StartupExitPage
        Public Overrides Property PageName As String
            Get
                Return Language.Language.strStartupExit
            End Get
            Set
            End Set
        End Property

        Public Overrides Sub ApplyLanguage()
            MyBase.ApplyLanguage()

            chkSaveConsOnExit.Text = Language.Language.strSaveConsOnExit
            chkReconnectOnStart.Text = Language.Language.strReconnectAtStartup
            chkSingleInstance.Text = Language.Language.strAllowOnlySingleInstance
            chkProperInstallationOfComponentsAtStartup.Text =
                Language.Language.strCheckProperInstallationOfComponentsAtStartup
        End Sub

        Public Overrides Sub SaveSettings()
            MyBase.SaveSettings()

            My.Settings.SaveConsOnExit = chkSaveConsOnExit.Checked
            My.Settings.OpenConsFromLastSession = chkReconnectOnStart.Checked
            My.Settings.SingleInstance = chkSingleInstance.Checked
            My.Settings.StartupComponentsCheck = chkProperInstallationOfComponentsAtStartup.Checked
        End Sub

        Private Sub StartupExitPage_Load(sender As Object, e As EventArgs) Handles MyBase.Load
            chkSaveConsOnExit.Checked = My.Settings.SaveConsOnExit
            chkReconnectOnStart.Checked = My.Settings.OpenConsFromLastSession
            chkSingleInstance.Checked = My.Settings.SingleInstance
            chkProperInstallationOfComponentsAtStartup.Checked = My.Settings.StartupComponentsCheck
        End Sub
    End Class
End Namespace